src/index.ts

```diff
@@ -8,8 +8,8 @@
 * Copyright IBM Corp, 2019
 *
 */
-export * from "./api/AutoBundler";
-export * from "./api/Bundle";
-export * from "./api/BundlePart";
-export * from "./api/Manifest";
-export * from "./api/NodejsappBundlePart";
+export * from "./api/BundleContent/AutoBundler";
```

Comment:
I'm not sure about the directory name 'BundleContent' - suggest 'generate'.

Reply:
It's not just about generation... it could be used for a query interface in the future, or updating, or validation... naming it BundleContent seemed more sensible to me given where it might evolve to.